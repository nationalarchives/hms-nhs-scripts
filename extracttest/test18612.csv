data.text
18-2-1826
7-3-26
31-05-1826
26.10.1826
28 - 09 - 1827
10-010-1828
010-010-1828
10-010-828
