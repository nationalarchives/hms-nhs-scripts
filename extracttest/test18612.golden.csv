data.text
18-02-1826
07-03-1826
31-05-1826
26-10-1826
28-09-1827
10-10-1828
10-10-1828
10-010-828
